helloworld-mutual-ssl/src/test/java/org/jboss/as/quickstarts/helloworld_mutual_ssl/BasicRuntimeIT.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.as.quickstarts.helloworld_mutual_ssl;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * The very basic runtime integration testing.
+ * @author Prarthona Paul
+ * @author emartins
+ */
+public class BasicRuntimeIT {
+
+    private static final String DEFAULT_SERVER_HOST = "http://localhost:8080";
+
+    @Test
+    public void testHTTPEndpointIsAvailable() throws IOException, InterruptedException, URISyntaxException {
+        String serverHost = System.getenv("SERVER_HOST");
+        if (serverHost == null) {
+            serverHost = System.getProperty("server.host");
+        }
+        if (serverHost == null) {
+            serverHost = DEFAULT_SERVER_HOST;
+        }
+        final HttpRequest request = HttpRequest.newBuilder()
+                .uri(new URI(serverHost+"/"))
```

Comment:
I believe this should be                 
.uri(new URI(serverHost+"/HelloWorld"))

Reply:
I have tested it with both `.uri(new URI(serverHost+"/"))` and `.uri(new URI(serverHost+"/HelloWorld"))` and the former seems to pass the tests while the latter gives the following errors: 
```
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.217 s <<< FAILURE! - in org.jboss.as.quickstarts.helloworld_mutual_ssl.BasicRuntimeIT
[ERROR] testHTTPEndpointIsAvailable(org.jboss.as.quickstarts.helloworld_mutual_ssl.BasicRuntimeIT)  Time elapsed: 0.196 s  <<< FAILURE!
java.lang.AssertionError: expected:<200> but was:<500>
	at org.jboss.as.quickstarts.helloworld_mutual_ssl.BasicRuntimeIT.testHTTPEndpointIsAvailable(BasicRuntimeIT.java:57)
```
When I ran the quickstart following the README file, I saw that https://localhost:8443/helloworld-mutual-ssl redirects to https://localhost:8443/helloworld-mutual-ssl/HelloWorld, so I am not sure why the tests are giving 500 error. 
If you have any suggestions, please feel free to let me know. 
Or if leaving it at `.uri(new URI(serverHost+"/"))` works, then I can change it to that too. 
Thank you for reviewing the PR :)